pkgs/native_assets_builder/test_data/native_add_version_skew/ffigen.yaml

```
@@ -0,0 +1,20 @@
+# Run with `flutter pub run ffigen --config ffigen.yaml`.
+name: NativeAddBindings
+description: |
+  Bindings for `src/native_add.h`.
+
+  Regenerate bindings with `flutter pub run ffigen --config ffigen.yaml`.
+output: "lib/src/native_add_bindings_generated.dart"
+headers:
+  entry-points:
+    - "src/native_add.h"
+  include-directives:
+    - "src/native_add.h"
+preamble: |
+  // Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Nit
```suggestion
  // Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
```

This also shows a downside of yaml configs, you can't write `${DateTime.now().year}` ;)